app/models/reason_code.rb

```diff
@@ -2,7 +2,8 @@
 
 class ReasonCode < ApplicationRecord
   has_many :incidents
-  validates :identifier, :description, presence: true, uniqueness: true
+  validates :identifier, :description, presence: true
+  validates :identifier, uniqueness: true # description isn't unique, apparently
```

Comment:
What do you mean by apparently? This was requested? Or something from Hastus?

Reply:
I'd have to go back and look at the history of where these came from to be 100% sure, but I'm pretty sure it's a list configured by SATCO.

Reply:
Alright, cool